components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/DataFlowProcessingEngine.cs

```diff
@@ -165,56 +166,68 @@ public class DataFlowProcessingEngine : IProcessingEngine, IDisposable {
             double valueChangesPerSec = _messageTrigger.ValueChangesCount / totalDuration;
             double dataChangesPerSec = _messageTrigger.DataChangesCount / totalDuration;
             double sentMessagesPerSec = totalDuration > 0 ? _messageSink.SentMessagesCount / totalDuration : 0;
```

Comment:
add totalDuration > 0?

Reply:
This shows the speed in "msg/s". This is basically a check for division by zero